corrector.rb

```diff
@@ -0,0 +1,7 @@
+class Corrector
+  def correct_name(name)
+    corrected_name = name.capitalize
+    ccorrected_name.slice(0..10) if name.lentgh > 10
+    corrected_name
+  end
```

Comment:
- [ ] You have syntax errors in this method to address. I think you meant `length`, not `lentgh`. Kindly fix this.
- [ ] On the same line 4, I think you meant `corrected_name` (one 'c'), not `ccorrected_name` (double 'c'). Kindly fix that.
- [ ] Also, this method does not return a sliced string of 10 characters (see image). Kindly fix that. 
  
  **SUGGESTION** You need to update the `corrected_name` variable like so:
  ```
  corrected_name = corrected_name.slice(0..10) if name.length > 10
  ```

  ![ranjeet-10](https://user-images.githubusercontent.com/36542986/156153342-c8ea71b3-4237-46d9-b309-0a70313c2a54.jpg)